index.php

```diff
@@ -4,91 +4,108 @@
 include_once("../../assets/static_pages/khr_page_top.php");
 ?>
 
-<p>The OpenVX registry contains specifications of the core API, headers,
-  extensions, and related documentation.</p>
+<p>The OpenVX registry contains specifications of the core API, headers, extensions, and related documentation.</p>
 
 <h3>OpenVX API Specification, Headers, and Documentation</h3>
 
-<p>The current version of OpenVX is OpenVX 1.2.1:</p>
-<ul>
-  <li> OpenVX 1.2.1 specification (<a href="specs/1.2.1/OpenVX_Specification_1_2_1.html">HTML</a>,
-    <a href="specs/1.2.1/OpenVX_Specification_1_2_1.pdf">PDF</a>)
-    (updated November 27, 2018).</li>
-  <li> <a href="api/1.2/openvx-standard-headers-1.2.tar.bz2"> OpenVX 1.2 header
-    package </a> (updated October 20, 2017).</li>
-  <li> <a href="sample/openvx_sample_1.2.tar.bz2">OpenVX 1.2 sample code</a>
-    (updated January 17, 2018). </p>
-</ul>
+<p>The current version of OpenVX is OpenVX 1.3:</p>
+  <ul>
+    <li>OpenVX 1.3 specification (<a href="specs/1.3/OpenVX_Specification_1_3.html">HTML</a>,
```

Comment:
This link appears to be broken.  It either needs to be changed to "specs/1.3/html/OpenVX_Specification_1_3.html" or better yet, the file should be moved out of the html directory (removing that level of the path).  Also, a copy of the "katex" directory needs to be added as the same level as the spec html.  See "specs/1.2.1" for an example.